sdk/python/lib/pulumi/automation/_local_workspace.py

```diff
@@ -301,6 +301,9 @@ def list_stacks(self) -> List[StackSummary]:
 
     def install_plugin(self, name: str, version: str, kind: str = "resource") -> None:
         self._run_pulumi_cmd_sync(["plugin", "install", kind, name, version])
+        
+    def install_plugin_from_server(self, name: str, version: str, kind: str = "resource", server: str) -> None:
```

Comment:
Adding non default arguments are default arguments is bad, but I didn't want to break the API ordering here. Thoughts?